src/main/java/inventory/mysql/MHConsumer.java

```diff
@@ -0,0 +1,176 @@
+package inventory.mysql;
```

Comment:
This file setups the JAAS configuration before talking with MessageHub. Then it starts polling topic